pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -102,6 +102,8 @@ type managerImpl struct {
 	thresholdsLastUpdated time.Time
 	// whether can support local storage capacity isolation
 	localStorageCapacityIsolation bool
+	// podsEvicted is the set of pods (format: namespace/pod: bool) that we have already tried to evict
+	podsEvicted map[string]bool
```

Comment:
should we track pods by their UID or by their namespace/name pair?
In addition, do we want to use a set (k8s libs: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets) ?